LipParse/app/Main.hs

```diff
@@ -1,13 +1,18 @@
 module Main where
 
-import Parse 
+import           Parse
```

Comment:
is this right ?

Reply:
Yes, default in haskell due to qualified imports